cmd/bucket-notification-handlers.go

```diff
@@ -45,6 +45,9 @@ func (api objectAPIHandlers) GetBucketNotificationHandler(w http.ResponseWriter,
 	ctx := newContext(r, w, "GetBucketNotification")
 
 	defer logger.AuditLog(ctx, r)
+	for k, v := range extractReqParams(r) {
```

Comment:
2 different styles are used:
```
for k, v := range extractReqParams(r) {
	logger.GetReqInfo(ctx).SetTags(k, v)
}
```
vs
```
reqParams := extractReqParams(r)
for k, v := range reqParams {
	logger.GetReqInfo(ctx).SetTags(k, v)
}
```

In some cases, `reqParams` is needed to be defined for later use, but we can use the shorter version whenever there is no need.

Reply:
Some places we don't need to reuse reqParams some places we do.